<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #222831;
color: #ffffff;
font: 57 10pt &quot;Yu Gothic Medium&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="drawingArea">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFeelingLucky">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #885a89;</string>
    </property>
    <property name="text">
     <string>I'm feeling lucky</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radbtnEnableMouse">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #885a89;</string>
    </property>
    <property name="text">
     <string>Enable Mouse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Translation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>310</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Scaling:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SpinBoxTx">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 74, 73);</string>
    </property>
    <property name="minimum">
     <double>-600.000000000000000</double>
    </property>
    <property name="maximum">
     <double>600.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SpinBoxTy">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 74, 73);</string>
    </property>
    <property name="minimum">
     <double>-600.000000000000000</double>
    </property>
    <property name="maximum">
     <double>600.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SpinBoxAngle">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>300</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 74, 73);</string>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SpinBoxSx">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>410</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 74, 73);</string>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SpinBoxSy">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 74, 73);</string>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTranslate">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>160</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 215, 102);
color: #222831;</string>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotate">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>300</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 215, 102);
color: #222831;</string>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScale">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>430</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 215, 102);
color: #222831;</string>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TRS">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>160</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 215, 102);
color: #222831;</string>
    </property>
    <property name="text">
     <string>TRS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRTS">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>300</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 215, 102);
color: #222831;</string>
    </property>
    <property name="text">
     <string>RTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSRT">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>430</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 215, 102);
color: #222831;</string>
    </property>
    <property name="text">
     <string>SRT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColor">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>560</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 159, 183);</string>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>560</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 159, 183);</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Ty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>774</x>
      <y>310</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>420</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>470</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>640</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0e1c36;</string>
    </property>
    <property name="text">
     <string>21132_Shreyash</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>660</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0e1c36;</string>
    </property>
    <property name="text">
     <string>A9. Basic Transformations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
